site/content/entry/citra-progress-report-2023-q1/index.md

```diff
@@ -0,0 +1,409 @@
+
+{{< sidebyside "image" ""
+    "marioOld.png=Citra, Citra and... Citra?"
+    "marioNew.png=We're all here now! Let's play!" >}}
+
+# Frontend Improvements
+
+### Quick screenshot ([#6025](https://github.com/citra-emu/citra/pull/6025)) by [Z11-V](https://github.com/Z11-V)
+
+An arguably simpler way of taking screenshots in game, without taking you away from the action to name and save them! Use the Capture Screenshot hotkey to quickly save them to the screenshots folder in the [Citra User Directory](https://citra-emu.org/wiki/user-directory/).
+
+### Added Midnight Theme ([#6030](https://github.com/citra-emu/citra/pull/6030)) by [Daisouji](https://github.com/Daisouji)
+
+Sick of using just the default Light and Dark themes? Inspired by the yuzu theme of the same name, “Midnight Blue” and “Midnight Blue Colorful” are available in Citra to give you even more customizability than before!
+
+{{< figure src="midnight.png"
+    title="Ooh! Shiny new colors!" >}}
+
+### Citra SDL: graceful shutdown on application shutdown ([#6102](https://github.com/citra-emu/citra/pull/6102)) by [ian-h-chamberlain](https://github.com/ian-h-chamberlain)
+
+When closing Citra SDL, the command line counterpart to our normal GUI version that most people will be used to, the window would remain open unintentionally which required a forceful shutdown to close properly. This would result in cleanup tasks, such as video dumping, failing. Which isn’t very nice if you’ve just spent a long time working on a video just to lose it! 
+
+By implementing a graceful shutdown, the window will close normally and a standard cleanup can be utilized. This results in better closing practices for the SDL version of Citra, and ensures that any data is not lost either!
+
+## Separate Windows ([#6177](https://github.com/citra-emu/citra/pull/6177), [#6198](https://github.com/citra-emu/citra/pull/6198)) by [epicboy](https://github.com/ameerj)
+
+Another very long awaited feature, Separate Windows, has arrived in Citra!
+
+Since the conception of Citra in 2014 (yes, that long ago!) users had been asking for a way to split the two screens of the 3DS shown in Citra into two separate windows. This would allow you to move the two screens around to wherever you want on your PC, and would allow for them to be used on two separate monitors too! Unfortunately, no one was able to do this for a very long time, until [epicboy](https://github.com/ameerj) came along…
+
+The groundwork for this change had already been put in place with a PR [jroweboy](https://github.com/jroweboy) made for [vsync presentation](https://github.com/citra-emu/citra/pull/4940) back in 2019. The vsync presentation PR was created as a way to force enable vsync for everyone from within Citra. In the past, this was disabled by default. 
+For those who force enabled vsync in their driver settings, Citra wasn’t able to disable vsync, which resulted in inconsistent frame limiting. 
```

Comment:
Someone said the groundwork for separate windows started on that PR on the original doc (no idea who because all the changes are anonymous) and I just trusted that it was correct as I've not got enough technical knowledge on that side of stuff.